spring-cloud-gcp-data-spanner/src/test/java/org/springframework/data/spanner/core/mapping/BasicSpannerPersistentEntityTests.java

```diff
@@ -16,30 +16,41 @@
 
 package org.springframework.data.spanner.core.mapping;
 
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.Set;
-
-import org.junit.Assert;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
 import org.junit.Test;
-import org.junit.runner.RunWith;
 
 import org.springframework.data.annotation.Id;
 import org.springframework.data.util.ClassTypeInformation;
-import org.springframework.test.context.junit4.SpringRunner;
 
 /**
  * @author Chengyuan Zhao
  */
-@RunWith(SpringRunner.class)
 public class BasicSpannerPersistentEntityTests {
 
 	@Test
 	public void testTableName() {
 		BasicSpannerPersistentEntity<TestEntity> entity = new BasicSpannerPersistentEntity<>(
 				ClassTypeInformation.from(TestEntity.class));
 
-		Assert.assertEquals("custom_test_table", entity.tableName());
+		MatcherAssert.assertThat(entity.tableName(), Matchers.is("custom_test_table"));
```

Comment:
nit: I'd prefer static imports for assertThat() and is()

Reply:
I had that, but the checkstyle didn't let me import static those

Reply:
@artembilan any chance that we can loosen this rule up for Hamcrest/Junit classes?

Reply:
Sure! Just fix this Checkstyle rule:
```
<module name="AvoidStaticImport">
			<property name="excludes"
				value="org.hamcrest.CoreMatchers.*,
				org.hamcrest.Matchers.*,
				org.hamcrest.collection.IsCollectionWithSize.*,
				org.junit.Assert.*,
				org.junit.Assume.*,
				org.mockito.Mockito.*,
				org.mockito.BDDMockito.*,
				org.mockito.AdditionalAnswers.*,
				org.mockito.ArgumentMatchers.*,
				org.mockito.hamcrest.MockitoHamcrest.*" />
		</module>
```

Reply:
Great!! Thank you, perfect!